checker/tests/resourceleak/MultipleMethodParamsMustCallAliasTest.java

```diff
@@ -0,0 +1,62 @@
+import java.io.*;
+import java.net.*;
+import org.checkerframework.checker.calledmethods.qual.*;
+import org.checkerframework.checker.mustcall.qual.*;
+import org.checkerframework.common.returnsreceiver.qual.*;
+
+class MultipleMethodParamsMustCallAliasTest {
+
+  void testMultiMethodParamsCorrect(@Owning InputStream in1, @Owning InputStream in2)
+      throws IOException {
+
+    ReplicaInputStreams r = new ReplicaInputStreams(in1, in2);
+
+    r.close();
+  }
+
+  // :: error: required.method.not.called
+  void testMultiMethodParamsWrong1(@Owning InputStream in1, @Owning InputStream in2)
```

Comment:
Can you add a variant of this test that calls both `in1.close()` and `in2.close()` on all paths? I think that ought to be verifiable and not issue any errors.

Reply:
I've added two more tests to call close on in1 and in2 on all paths.